caroline.yml

```diff
@@ -0,0 +1,18 @@
+scheme: "caroline"
+author: "ed"
```

Comment:
Maybe add a link to Ed’s other repo or something? Not necessary but could be nice to have.

Reply:
Good ideia!